pinc/Stage.inc

```diff
@@ -10,7 +10,29 @@
 
 include_once($relPath.'Activity.inc');
 
-$Stage_for_id_ = [];
+/*
+ * A singleton container class for accessing Stage objects.
+ * See the Activities class definition for more information.
+ */
+class Stages extends Activities
```

Comment:
It would be simpler not to extend Activities but have Activities, Stages, Pools and Rounds to be independent classes. Then when the different activities are created each one will get the appropriate activities set in it: Stages will get all the Rounds and Pools and Activities will get everything. When you create a Round it creates its parent Stage which will put the Round into its Stages class and so on.